private/buf/cmd/buf/command/beta/convert/convert.go

```diff
@@ -214,3 +221,19 @@ func inverseEncoding(encoding bufconvert.MessageEncoding) (bufconvert.MessageEnc
 		return 0, fmt.Errorf("unknown message encoding %v", encoding)
 	}
 }
+
```

Comment:
I can't comment a few lines up, but I'm a little surprised the `inverseEncoding` function is persisting to stable, I would have thought we removed this for stable, worth a discussion.

Reply:
https://github.com/bufbuild/buf/issues/1432#issuecomment-1255555035

input encoding and output encoding can be explicitly set. inverse encoding only gets called if output doesn't have a `-#format` option (or the format can't be inferred by the filename)